doc/index.rst

```diff
@@ -2,6 +2,219 @@
+   INFO:ceph-daemon:Setting orchestrator backend to ssh...
+   INFO:ceph-daemon:Adding host 192-168-1-101.tpgi.com.au...
+   INFO:ceph-daemon:Enabling the dashboard module...
+   INFO:ceph-daemon:Waiting for the module to be available...
+   INFO:ceph-daemon:Generating a dashboard self-signed certificate...
+   INFO:ceph-daemon:Creating initial admin user...
+   INFO:ceph-daemon:Fetching dashboard port number...
+   INFO:ceph-daemon:Ceph Dashboard is now available at:
+   URL: https://192-168-1-101.tpgi.com.au:8443/
+   User: admin
+   Password: oflamlrtna
+   INFO:ceph-daemon:You can access the Ceph CLI with:
+   sudo ./ceph-daemon shell -c ceph.conf -k ceph.keyring
+   INFO:ceph-daemon:Bootstrap complete.
+   END QUOTED TEXT
+
+
+Second Node
+-----------
+
+#. Install a recent, supported Linux distribution on a second computer.
+#. Install podman, docker, and python::
+
+    $ sudo dnf install podman docker python
+#. Turn on ssh on node 2:::
+
+    $ sudo systemctl start sshd
+    $ sudo systemctl enable sshd
+#. Create a file on node 2 that will hold the ceph public key::
+
+    $ sudo mkdir -p /root/.ssh
+    $ sudo touch /root/.ssh/authorized_keys
+#. Copy the public key from node 1 to node 2::
+
+    [node 1] $ sudo cat ceph.pub | ssh root@192.168.1.102 tee -a /root/.ssh/authorized_keys
+
+Third Node
+----------
+#. Install a recent, supported Linux distribution on a third computer.
+
+#. Install podman, docker, and python::
+
+    $ sudo dnf install podman docker python
+#. Turn on ssh on node 3::
+
+    $ sudo systemctl start sshd
+    $ sudo systemctl enable sshd
+#. Create a file on node 3 that will hold the ceph public key::
+
+    $ sudo mkdir -p /root/.ssh
+    $ sudo touch /root/.ssh/authorized_keys
+
+#. Copy the public key from node 1 to node 3::
+
+    [node 1] $ sudo cat ceph.pub | ssh root@192.168.1.103 tee -a /root/.ssh/authorized_keys
+#. On node 1, issue the command that adds node 3 to the cluster::
+
+    [node 1] $ sudo ceph orchestrator host add 192.168.1.103
+
+
+Creating Two More Monitors
+--------------------------
+
+#. Set up a Ceph monitor on node 2 by issuing the following command on node 1.
+   This command took six minutes to complete execution. So: wait::
+
+   [node 1] $ sudo ceph orchestrator mon update 2 192.168.1.102:192.168.1.0/24
+   [sudo] password for user:
+   ["(Re)deployed mon 192.168.1.102 on host '192.168.1.102'"]
+   [user@192-168-1-101 ~] $ \ 
+#. Set up a Ceph monitor on node 3 by issuing the following command on node 1::
+
+   [node 1] $ sudo ceph orchestrator mon update 3 192.168.1.103:192.168.1.0/24
+   [sudo] password for user:
+   ["(Re)deployed mon 192.168.1.103 on host '192.168.1.103'"]
+   [user@192-168-1-101 ~]$
+
+
+Creating OSDs
+-------------
+
+Creating an OSD on the First Node
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Use a command of the following form to create an OSD on node 1::
+
+   [node 1@192-168-1-101]$ sudo ceph orchestrator osd create 192-168-1-101:/dev/sdb1
```

Comment:
should use persistent paths for disks, i.e. /dev/disk/by-id/ata-WDC_WDS200T2B0A-00SM50_182294800028 rather than /dev/sdb1

Reply:
This change is in. Persistent paths are now specified by the docs.

Reply:
missed this line, the other two are updated

Reply:
Got it.